src/universal/modules/summary/containers/MeetingSummary/MeetingSummaryContainer.js

```diff
@@ -99,20 +100,50 @@ const mapStateToProps = (state, props) => {
   };
 };
 
-const MeetingSummaryContainer = (props) => {
-  return <MeetingSummary {...props} />;
-};
+@requireAuth
+@connect(mapStateToProps)
+export default class MeetingSummaryContainer extends Component {
+  static propTypes = {
+    actionCount: PropTypes.number,
+    agendaItemsCompleted: PropTypes.number,
+    dispatch: PropTypes.func,
+    meetingNumber: PropTypes.number,
+    projectCount: PropTypes.number,
+    teamId: PropTypes.string,
+    teamMembers: PropTypes.array,
+    teamName: PropTypes.string
+  };
+
+  componentWillMount() {
+    this.state = {segmentEventTracksSent: false};
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const {segmentEventTracksSent} = this.state;
+    const {dispatch, meetingNumber} = nextProps;
+    if (!meetingNumber || segmentEventTracksSent) return;
+    /*
+     * Track meeting completitions by idenity.
+     *
+     * N.B. it is ok if these are sent as dupes, i.e. when viewed from
+     * meeting history.
+     */
+    dispatch(segmentEventTrack('Meeting Completed', { meetingNumber }));
```

Comment:
Hmmm, if this is only a quick & dirty temporary thing no worries, but I'd love to move this to the server. We could simplify it a little: `if (!this.props.meetingNumber && nextProps.meetingNumber) dispatch(....` to get rid of the component state, but leaving this as is will push us to kill it that much faster :smile: